projects/packages/blaze/src/class-blaze.php

```diff
@@ -123,7 +123,7 @@ public static function enable_blaze_menu() {
 				__( 'Advertising', 'jetpack-blaze' ),
 				'manage_options',
 				'https://wordpress.com/advertising/' . $domain,
-				null,
+				null, // @phan-suppress-current-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. TODO add link with Trac issue.
```

Comment:
I found an existing trac issue for this. 🙂
```suggestion
				null, // @phan-suppress-current-line PhanTypeMismatchArgumentProbablyReal -- Core should ideally document null for no-callback arg. https://core.trac.wordpress.org/ticket/52539
```

Reply:
I created https://github.com/Automattic/jetpack/pull/38075 to update the other instances of this comment currenty in the monorepo.